### PR TITLE
Fix change in jplhorizons RA/Dec column headers  with the January 21, 2020 update

### PR DESCRIPTION
**Pull request by talister**
RA, Dec, column names changed in the Version 4.67 HORIZONS update. This PR adds the new ones but leaves the old ones - maybe they could go now ? Addresses #1619

### CODE REVIEW

File: astroquery/jplhorizons/__init__.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ class Conf(_config.ConfigNamespace):
                    'phasecoeff': ('phasecoeff', 'mag/deg'),
                    'solar_presence': ('solar_presence', '---'),
                    'flags': ('flags', '---'),
+                   'R.A._(ICRF)': ('RA', 'deg'),
+                   'DEC_(ICRF)': ('DEC', 'deg'),
                    'R.A._(ICRF/J2000.0)': ('RA', 'deg'),
```
Comment (bsipocz) [inline astroquery/jplhorizons/__init__.py]: both of these are now present, or it's rather a rename of `R.A._(ICRF/J2000.0)` to `R.A._(ICRF)`?

**Comment by bsipocz** [comment]
cc @mommermi

File: astroquery/jplhorizons/__init__.py (hunk 1 above)
Comment (talister) [reply astroquery/jplhorizons/__init__.py]: I think the latter but there is no official word in the News or Documentation and the underlying telnet interface still mentions `ICRF/J2000.0` as the reference frame. I think it's safe to have both, but will defer to @mommermi judgement

**Comment by bsipocz** [comment]
Hmm. Looking at the remote test failures, there are a ton of other issues. Would you be interested in exploring fixes to those here, or rather prefer to wait for the separate fixes? I'm afraid we won't ship a release until it's all done.

**Comment by talister** [comment]
I can try and take a look at them. How can I see the remote test results or run them myself ?

**Comment by bsipocz** [comment]
`python setup.py test -P jplhorizons -R` will run tests only for the jplhorizons module, -R runs the remote ones, too.

With a quick look, the issue is the same as before with RA/DEC, there are new columns that raise KeyErrors `KeyError: 'vis.'`

Here is the full list of columns returned at the problematic part:
```
> /Users/bsipocz/munka/devel/worktrees/astroquery/changelog/astroquery/jplhorizons/core.py(1247)_parse_horizons()
   1245         rename = []
   1246         for col in data.columns:
-> 1247             data[col].unit = column_defs[col][1]
   1248             if data[col].name != column_defs[col][0]:
   1249                 rename.append(data[col].name)

ipdb> data                                                                                                                                            
/usr/local/lib/python3.7/site-packages/ipykernel/iostream.py:14: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  from imp import lock_held as import_lock_held
<Table masked=True length=1>
targetname Date__(UT)__HR:MN:SC.fff Date_________JDUT    H       G    solar_presence ... Tru_Anom L_Ap_Hour_Ang   phi   PAB-LON  PAB-LAT
   ---               ---                    d           mag     ---        ---       ...                                                
   str7             str24                float64      float64 float64      str1      ... float64      int64     float64 float64  float64
---------- ------------------------ ----------------- ------- ------- -------------- ... -------- ------------- ------- -------- -------
   1 Ceres 2000-Jan-01 00:00:00.000         2451544.5     3.4    0.12                ...   7.1181            -- 22.5692 172.8356 11.3482
ipdb> data.colnames                                                                                                                                   
['targetname', 'Date__(UT)__HR:MN:SC.fff', 'Date_________JDUT', 'H', 'G', 'solar_presence', 'flags', 'R.A._(ICRF)', 'DEC_(ICRF)', 'R.A._(a-app)', 'DEC_(a-app)', 'dRA*cosD', 'd(DEC)/dt', 'Azi_(a-app)', 'Elev_(a-app)', 'dAZ*cosE', 'd(ELV)/dt', 'X_(sat-prim)', 'Y_(sat-prim)', 'SatPANG', 'L_Ap_Sid_Time', 'a-mass', 'mag_ex', 'APmag', 'S-brt', 'Illu%', 'Def_illu', 'ang-sep', 'vis.', 'Ang-diam', 'Ob-lon', 'Ob-lat', 'Sl-lon', 'Sl-lat', 'SN.ang', 'SN.dist', 'NP.ang', 'NP.dist', 'hEcl-Lon', 'hEcl-Lat', 'r', 'rdot', 'delta', 'deldot', '1-way_down_LT', 'VmagSn', 'VmagOb', 'S-O-T', '/r', 'S-T-O', 'T-O-M', 'MN_Illu%', 'O-P-T', 'PsAng', 'PsAMV', 'PlAng', 'Cnst', 'TDB-UT', 'ObsEcLon', 'ObsEcLat', 'N.Pole-RA', 'N.Pole-DC', 'GlxLon', 'GlxLat', 'L_Ap_SOL_Time', '399_ins_LT', 'RA_3sigma', 'DEC_3sigma', 'SMAA_3sig', 'SMIA_3sig', 'Theta', 'Area_3sig', 'POS_3sigma', 'RNG_3sigma', 'RNGRT_3sig', 'DOP_S_3sig', 'DOP_X_3sig', 'RT_delay_3sig', 'Tru_Anom', 'L_Ap_Hour_Ang', 'phi', 'PAB-LON', 'PAB-LAT']
```

File: astroquery/jplhorizons/__init__.py (hunk 1 above)
Comment (mommermi) [reply astroquery/jplhorizons/__init__.py]: I'm fine with keeping both definitions and if it's only to keep the tests alive. There should be no issues with name conflicts.

**Comment by mommermi** [comment]
Not sure what the remaining build errors means - maybe it just needs a restart?

**Comment by talister** [comment]
@mommermi @bsipocz one issue is that the static capture of HORIZONS output in the offline tests directory is now out of date. What is the policy for updating these ? Should there be one file which is current and what the service produces now or should there be a second set of tests for the new output format and the old one is kept ?

**Comment by mommermi** [comment]
You can just overwrite the static data file with a current one and update the values that are checked against accordingly. Let me know if you need help with this.

**Comment by bsipocz** [comment]
> Not sure what the remaining build errors means - maybe it just needs a restart?

the travis matrix needs updating as the combination is not supported, that's beyond the scope here.

**Comment by bsipocz** [comment]
Perfect, thank you @talister!

**Comment by bsipocz** [comment]
Would you mind adding a single liner into the changelog to say the module has been updated to be compatible with the upstream changes? If you add a `[skip ci]` text at the end of the commit message, the CI is not being triggered by it.

Then I'll merge this once travis is green for the updated file.

**Comment by talister** [comment]
> Would you mind adding a single liner into the changelog to say the module has been updated to be compatible with the upstream changes? If you add a `[skip ci]` text at the end of the commit message, the CI is not being triggered by it.
> 
> Then I'll merge this once travis is green for the updated file.

Into the `CHANGES.rst` file in the root directory under the `jplhorizons` heading ?

**Comment by bsipocz** [comment]
> Into the CHANGES.rst file in the root directory under the jplhorizons heading ?

yes, exactly, under the `0.4 (unrelesed)` part.

**Comment by bsipocz** [comment]
Thank you @talister!

**Comment by brycebolin** [comment]
Thank you for the fix!

**Pull request merged**